fancy-explosions/BangazonSite
# Deletes a Users entire shopping cart order

# Description
Delete button at very bottom of shopping cart page deletes and entire order with all the order products as well. If shopping cart is empty and you click the view cart link it will take you to the product index page.



Fixes Issue # https://github.com/fancy-explosions/BangazonSite/issues/8

## Type of change

Please delete options that are not relevant.



# Testing Instructions

git fetch --all
git checkout cm-order-delete
make sure you have products in your shopping cart
Click delete button at very bottom of page and ensure entire order is deleted
after order is deleted if you click the view cart link it should redirect you to the products index page.



# Checklist:
